cdc/http_handler.go

```diff
@@ -119,6 +123,33 @@ func (s *Server) handleRebanlanceTrigger(w http.ResponseWriter, req *http.Reques
 		return
 	}
 	changefeedID := req.Form.Get(APIOpVarChangefeedID)
-	err = s.owner.TriggerRebanlance(changefeedID)
-	handleOwnerResp(w, err)
+	s.owner.TriggerRebanlance(changefeedID)
+	handleOwnerResp(w, nil)
+}
+
+func (s *Server) handleMoveTable(w http.ResponseWriter, req *http.Request) {
+	if req.Method != http.MethodPost {
+		writeError(w, http.StatusBadRequest, errors.New("this api only supports POST method"))
+		return
+	}
+
+	if s.owner == nil {
+		handleOwnerResp(w, concurrency.ErrElectionNoLeader)
+	}
+
+	err := req.ParseForm()
+	if err != nil {
+		writeInternalServerError(w, err)
+		return
+	}
+	changefeedID := req.Form.Get(APIOpVarChangefeedID)
+	to := req.Form.Get(APIOpVarTargetCaptureID)
+	tableIDStr := req.Form.Get(APIOpVarTableID)
```

Comment:
Better to validate inputs first.

Reply:
I validate the inputs here: https://github.com/pingcap/ticdc/pull/593/files#diff-1956ba2b6a217651590eb6f946442117R444
If the user inputs incorrectly, we should provide better error info. but it's hard to validate the inputs here, because we can't access changefeed or capture directly. This will cause the data race problems.
maybe we should add a TODO here?

Reply:
Just make sure `from` and `to` are valid UUIDv4 string.